csharp/ql/integration-tests/all-platforms/blazor/remoteFlowSource.ql

```diff
@@ -0,0 +1,7 @@
+import semmle.code.csharp.security.dataflow.flowsources.Remote
```

Comment:
What is the motivation for using integration tests instead of normal QL tests? I find it quite difficult to run 
(specific) integration tests locally...

Reply:
This was mostly because the test was originally from Tamás's WIP PR. And I wasn't sure about testing Blazor components as a QL test (I'm unsure if I'll need to have a full project). I'll experiment with moving it.

You are correct about the integration test being a more complicated way to run the test locally.

Reply:
If it's not easy to do, then let's do only the integration test.